app/src/main/java/mozilla/lockbox/store/AccountStore.kt

```diff
@@ -96,6 +97,18 @@ open class AccountStore(
             }
             .addTo(compositeDisposable)
 
+        // Moves credentials from the AccountStore, into the DataStore.
+        syncCredentials.map {
```

Comment:
ooo

we don't typically have stores dispatch actions?
would it make more sense to have the datastore subscribe to the account store, or ?

Reply:
I think stores dispatching actions is fine. Subscribing to others makes me more queasy. Let's discuss.